pgoapi/poke_utils.py

```diff
@@ -30,6 +110,15 @@ def parse_game_master():
             game_master[int(line[0])] = pokemon_data
     return game_master
 
+def pokemon_lvl(DEFINE_POKEMON_LV, pokemon):
+    if DEFINE_POKEMON_LV == "CP":
+        return pokemon.cp
+    elif DEFINE_POKEMON_LV == "IV":
+        return pokemon.iv
+    elif DEFINE_POKEMON_LV == "CP*IV":
+        return pokemon.cp * pokemon.iv
+    elif DEFINE_POKEMON_LV == "CP+IV":
+        return pokemon.cp + pokemon.iv
```

Comment:
this is redundant, this functionality was added to the pokemon class and is called score now, since we have an actual level. i suggest you try to rebase your PR